src/shapes/Object/Object.ts

```diff
@@ -1454,7 +1463,10 @@ export class FabricObject<
    * @return {Boolean}
    */
   isType(...types: string[]) {
```

Comment:
should we deprecate this also??

Reply:
not part of this PR.
isType is not directly related to the incident. nor `isType` is wrong per se.